frontend/app/services/IdentityService.scala

```diff
@@ -103,9 +99,7 @@ class IdentityService(identityApiProvider: => IdentityApi, implicit val ec: Exec
   }).valueOr(_ => false)
 
   def getFullUserDetails(user: IdMinimalUser)(implicit identityRequest: IdentityRequest): Future[IdUser] =
-    retry.Backoff(max = 3, delay = 2.seconds, base = 2){ () =>
-      getUser(user).value
-    }.map(_.fold({ errorMessage =>
+    getUser(user).value.map(_.fold({ errorMessage =>
```

Comment:
do you know if there was any reason these calls had a retry ?

Reply:
Presumably both of these calls were often unreliable at some stage. Given that the retries were added some time ago, and this app is now much lower traffic, I'm prepared to remove these and then keep an eye on how things fare in production unless anyone strongly disagrees?

Reply:
I think it's fine but I would wait to merge this on Monday

Reply:
Agreed 👍

Reply:
oh yes sorry I found these deps via snyk and clearly didn't actully push them out.  Thanks for doing that. I found by following the deps that we weren't actually using the vulnerable library, so I wasn't concerned about an actual security issue.